src/module.c

```diff
@@ -9368,6 +9501,16 @@ void moduleRegisterCoreAPI(void) {
     REGISTER_API(GetContextFromIO);
     REGISTER_API(GetKeyNameFromIO);
     REGISTER_API(GetKeyNameFromModuleKey);
+    REGISTER_API(GetDbFromModuleKey);
+    REGISTER_API(GetDbFromIO);
```

Comment:
a suggestion: change to `GetDbIdFromIO`, just like `GetKeyNameFromIO`

Reply:
Okay, modified ：）